src/pyodide/internal/setupPackages.ts

```diff
@@ -31,26 +31,32 @@ export const STDLIB_PACKAGES: string[] = Object.values(LOCKFILE.packages)
 // `folder/file.txt` -> `["folder", "file.txt"]
 export type FilePath = string[];
 
+function createTarFs() {
```

Comment:
I don't really appreciate the fact that we now have two functions called `createTarFs()`. Maybe this one could be `makeEmptyTarFSRoot()`? Or you could just use `structuredClone()` in the `SitePackagesDir` constructor, but maybe factoring it out is better.

Reply:
Oh I didn't even realise that a function with this name already existed. I'll name it something else.